docs/managing-data-quality/monitoring-failed-events/api/index.md

```diff
@@ -1,6 +1,7 @@
 ---
 title: "Accessing failed event aggregates via the API"
 date: "2020-06-12"
+sidebar_label: "Using the API"
 sidebar_position: 3000
 ---
 
```

Comment:
This reusable block says "Note" which I assume is meant to be an admon?

Reply:
Oh crap I probably need to kill this block altogether